src/App.js

```diff
@@ -2,12 +2,12 @@ import Header from "./shared/components/header/header";
 import Footer from "./shared/components/footer/footer";
 import "./global.css";
 import Home from "./pages/home/home";
+import ProductListing from "./pages/productListing/ProductListing";
 function App() {
   return (
     <div className="App">
-      <Header />
-      <Home />
-      <Footer />
+      {/* <Home /> */}
```

Comment:
Can remove this commented line of code if not required or can keep it depending on upcoming refactors?

Reply:
that code doesn't need to be removed if I will use routing . Hence I have kept it there.